src/main/java/org/cy3sbml/uniprot/UniprotAccess.java

```diff
@@ -75,50 +77,60 @@ public static String uniprotHTML(String accession){
         UniProtEntry entry = UniprotCache.getUniProtEntry(accession);
         if (entry != null) {
             String uniProtId = entry.getUniProtId().toString();
-            text += String.format(
-                    "\t<a href=\"https://www.uniprot.org/uniprot\"><img src=\"./images/logos/uniprot_icon.png\" title=\"Information from UniProt\"/></a>&nbsp;&nbsp;\n" +
-                            "\t<a href=\"https://www.uniprot.org/uniprot/%s\"><span class=\"identifier\">%s</span></a> (%s)<br />\n", accession, accession, uniProtId);
-
+            text += MessageFormat.format(
+                    "\t<a href=\"" + UNIPROT_URL + "\"><img src=\"./images/logos/uniprot_icon.png\" title=\"Information from UniProt\"/></a>&nbsp;&nbsp;\n" +
+                            "\t<a href=\"" + UNIPROT_URL + "/{0}\"><span class=\"identifier\">{1}</span></a> ({2})<br />\n",
+                    accession,
+                    accession,
+                    uniProtId
+            );
```

Comment:
What is the reason to use `MessageFormat` with `String` concatination? `UNIPROT_URL` can also be inserted through variable replacement. Also, should we better collect such longer text passages in a file and read it in so that we can separate out the source code from generated HTML fragments? In doing so, we might be able to maintain both independently and change them more easily. It would be possible to change the HTML templates without creating a new release, just by swapping the HTML files within the resource directory inside of a JAR file.